### PR TITLE
Refactor Database Naming

### PR DESCRIPTION
**Pull request by g-luo**
getProperty("documents") --> getProperty("docHashes")
User.getDocs() --> User.getDocHashes()

This makes naming more consistent, as any String that uniquely identifies a document is known as a "document hash". We implicitly differentiate between unique String identifiers as "hashes" and Long identifiers as "ids".

### CODE REVIEW

File: project/src/main/java/com/google/sps/Models/Database.java [hunk 1]
```diff
@@ -156,10 +160,40 @@ public static Document getDocumentByHash(String hash) {
...
     }
 
-    ArrayList<Long> userIDs = getListProperty(docEntity, "userIDs");
+    Document doc = getDocumentByHash(hash);
+    ArrayList<Long> userIDs = doc.getUserIDs();
+    //ArrayList<Long> userIDs = getListProperty(docEntity, "userIDs");
     return userIDs;
   }
 
+  public static ArrayList<Long> getDocumentEditors(String hash) {
```
Comment (max-friedman) [inline project/src/main/java/com/google/sps/Models/Database.java]: Should these permission changes be split into another branch/PR? Not sure what this has to do with naming refactoring

**Review by max-friedman — COMMENTED** [review]
Looks to me like maybe changes from 2 branches are in this PR, maybe try to split it up if you can

File: project/src/main/java/com/google/sps/Models/Database.java (hunk 1 above)
Comment (g-luo) [reply project/src/main/java/com/google/sps/Models/Database.java]: this was forked off the permissions branch; they should be de-linked now

**Review by tin0d0 — APPROVED** [review]
lgtm

**Review by frankpzn — APPROVED** [review]

**Review by max-friedman — APPROVED** [review]
lgtm

**Review by HidyHan — APPROVED** [review]

**Pull request merged**